blark/iec.lark

```diff
@@ -251,6 +253,7 @@ GENERIC_TYPE_NAME: "ANY"
 ?enumerated_type_name: IDENTIFIER
 ?array_type_name: IDENTIFIER
 ?structure_type_name: IDENTIFIER
+                    | DOTTED_IDENTIFIER
```

Comment:
Referencing back to your example: `_c_epoch : dot.dateTime_t := (dateTime := DT#1970-1-1-0:0:0, uSec := 0);`

That part looks good, but investigating where else `structure_type_name` is used reveals a potential problem:
https://github.com/engineerjoe440/blark/blob/ecffb0a5ebe4fa2bb5da4f701dd3b6a8932d5a21/blark/iec.lark#L342
This means we could start defining structures with dotted identifiers as well. I don't _think_ that's syntactical.

Reply:
Hmm... that's a good point. You're right! 😕

Reply:
Ok... I think I've got a fix now that should meet both goals in my latest commit. Let me know what you think!

Reply:
Changes look good - thanks!